src/lxc/tools/lxc_autostart.c

```diff
@@ -491,11 +491,15 @@ int main(int argc, char *argv[])
 
 	}
 
-	/* clean up any lingering detritus */
+	/* clean up any lingering detritus, if container exists here
+	 * then it must have failed to start.
+	 */
+	failed = 0;
```

Comment:
I initialized at declaration time to try and fit in with the current code.  I _really_ wanted the initialization here so it is close to the usage, especially since all we do is increment the var it seems cleaner to me to have the initialization close to the code that uses it.  I'm happy to put the initialization where ever though if you have a preference.

thanks

Reply:
It wast the initialization of `i` at declaration time that threw me, hence I initialized `failed` in similar fashion.